test/dub_package/.gitignore

```diff
@@ -1,6 +1,13 @@
+# Ignore all binary files (files without an extension)
```

Comment:
As hopefully more examples will be added, ignoring all binary files in this directory avoids frequent updates of this file.

Reply:
I think a cleaner solution would be put all dub artifacts in `generated/` as well, using `copyFiles` (see http://code.dlang.org/package-format?lang=sdl).